### DISCUSSION TITLE
MediaElement - Lock Screen Player

### DISCUSSION THREAD

**Discussion by natarajgandhi** [CommunityToolkit/Maui#1667] [General]
We are using MediaElement on our project and we are loving it. It is very stable and plays audio and video very well on Android and IOS (we support these two platforms on our app).

However, we are trying to implement a lock screen player when the user switches to audio mode. Out of the box, the mediaelement plugin doesnt seem to support a lock screen player.

Could you please help point me in the right direction on how we can get the lock screen player implemented while using the MediaElement control on our pages?

Thanks!

**Comment by ne0rrmatrix** [comment]
Working on it here: https://github.com/CommunityToolkit/Maui/pull/1782

**Comment by TheCodeTraveler** [comment]
Added in https://github.com/CommunityToolkit/Maui/pull/1782
